README.md

```diff
@@ -327,8 +359,8 @@ Compiling on Windows:
 	- Select your compiler
 	- It will warn about missing stuff, ignore that at this point. (later don't)
 	- Make sure the configuration is as follows
-	  (note that the versions may differ for you):
-	-----------------
+	### (note that the versions may differ for you):
```

Comment:
This is not supposed to be a title. The dashes should be replaced with a code block.